packages/styles/scss/components/_datacard.scss

```diff
@@ -14,7 +14,7 @@
 
       background: $brand-ilo-light-blue;
       border-bottom: px-to-rem(3px) solid $brand-ilo-ramp-blue;
-      padding: px-to-rem(48px) px-to-rem(52px);
+      padding: map-get($spacing, "spacing-12") map-get($spacing, "spacing-13");
```

Comment:
Hey @GGKapanadze I think we should conserve these as REM values, what do you think? Would just mean wrapping all of these map-gets() in the px-to-rem() function

Reply:
I'll create a new utility function for spacing like

```
padding: spacing("1")
```
and will wrap implementation inside

Reply:
@GGKapanadze excellent idea